docs/pages/base/api/badge-unstyled.json

```
@@ -23,13 +23,13 @@
   "slots": [
     {
       "name": "badge",
-      "description": "The component used to render the badge.",
+      "description": "The component for rendering the badge.",
```

The previous text made more sense to me. Alternatively "The component rendered as the bagde" could work. But let's ask @samuelsycamore what would work best here.

---

cc @siriwatknp

---

I think I would use this phrasing:

```suggestion
      "description": "The component that renders the badge.",
```

---

Made a change based on Sam's suggestion.

---

Could you regenerate the API docs with the updated text?

---

My bad. Done and rebased on top of master. I will merge https://github.com/mui/material-ui/pull/36328 first and then we can merge this one as well.